src/MCPServer/lib/linkTaskManagerDirectories.py

```diff
@@ -28,41 +28,27 @@
 import threading
 import traceback
 sys.path.append("/usr/lib/archivematica/archivematicaCommon")
-import databaseInterface
 import databaseFunctions
 from databaseFunctions import deUnicode
 from dicts import ReplacementDict
+sys.path.append("/usr/share/archivematica/dashboard")
+from main.models import StandardTaskConfig
 
 
 class linkTaskManagerDirectories(LinkTaskManager):
     def __init__(self, jobChainLink, pk, unit):
         super(linkTaskManagerDirectories, self).__init__(jobChainLink, pk, unit)
         self.tasks = []
-        sql = """SELECT * FROM StandardTasksConfigs where pk = '%s'""" % (pk.__str__())
-        c, sqlLock = databaseInterface.querySQL(sql)
-        row = c.fetchone()
-        if row == None:
-            print >>sys.stderr, "\nfind me\n"
-            traceback.print_exc(file=sys.stderr)
-            return None
-        while row != None:
-            print row
-            #pk = row[0]
-            filterFileEnd = deUnicode(row[1])
-            filterFileStart = deUnicode(row[2])
-            filterSubDir = deUnicode(row[3])
-            self.requiresOutputLock = deUnicode(row[4])
-            standardOutputFile = deUnicode(row[5])
-            standardErrorFile = deUnicode(row[6])
-            execute = deUnicode(row[7])
-            self.execute = execute
-            arguments = deUnicode(row[8])
-            row = c.fetchone()
-        sqlLock.release()
-        #if reloadFileList:
-        #    unit.reloadFileList()
-
-        #        "%taskUUID%": task.UUID.__str__(), \
+        stc = StandardTaskConfig.objects.get(id=str(pk))
+        filterFileEnd = stc.filter_file_end
+        filterFileStart = stc.filter_file_start
+        filterSubDir = stc.filter_subdir
+        self.requiresOutputLock = stc.requires_output_lock
```

Comment:
Should be fine! This was being fetched as an integer before, and Python treats `if 0/if 1` and `if False/if True` the same.